csrc/serde/fusion_cache.fbs

```diff
@@ -87,6 +87,7 @@ enum RecordType: int {
     ReductionSum,
     ReshapeOp,
     Scalar,
+    ScaledGroupedMmaOp,
```

Comment:
Naming doesn't seem to be consistent: ScaledGroupedMmaOp in fbs but GroupedMmaOp elsewhere?